Flick/Controllers/MediaViewController.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  MediaViewController.swift
+//  Flick
+//
+//  Created by Lucy Xu on 7/7/20.
+//  Copyright © 2020 flick. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class MediaViewController: UIViewController {
+
+    // MARK: - Private View Vars
+    private var mediaCardViewController: MediaCardViewController!
+    private let mediaImageView = UIImageView()
+    private let saveMediaButton = UIButton()
+    private var visualEffectView: UIVisualEffectView!
+
+    // MARK: - Private Data Vars
+    private var animationProgressWhenInterrupted: CGFloat = 0
+    private let buttonSize = CGSize(width: 72, height: 72)
```

Comment:
we used 44 for other buttons

Reply:
I checked the designs just now and it says 36? Maybe we can ask Cindy